src/main/java/com/epam/preprod/sirenko/entity/Food.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.preprod.sirenko.entity;
+
+import java.util.Objects;
+
+/**
+ * Food entity.
+ *
+ * @author E.Sirenko
+ **/
+public class Food extends Products {
+	private int amount;
```

Comment:
Maybe we should rename this field to quantity or weight to better describe behavior?

Reply:
renamed to weight